builder/pipeline.h

```diff
@@ -182,6 +184,14 @@ class func {
     compute_at_ = loop_id();
     return *this;
   }
+
+  void set_name(const std::string name) {
+    name_ = name;
+  }
+
+  const std::string& name() const {
```

Comment:
It looks like this isn't actually used anywhere, and it isn't plumbed into the `call_stmt`s... I think we should remove it unless we can properly support it. I haven't added this yet because I haven't figured out a way to do it that makes sense.

Reply:
I think in general it would be useful to have something like this, but agree that in the current form it's very hacky - removed.